wasm/js/animation_callback_handler.js

```diff
@@ -1,6 +1,41 @@
 // Manages a list of animation callbacks to be called in batch.
 // Override this.onAfterCallbacks to get a call once all animation callbacks have been invoked.
 function AnimationCallbackHandler() {
+    function FpsCounter() {
+        const div = document.createElement('div');
+        div.style.backgroundColor = 'black';
+        div.style.position = 'fixed';
```

Comment:
I'm finding that this div gets drawn mostly on top because we add it to the DOM late, but it can be placed underneath "position: absolute" elements, even if I set style.zOrder to something like 999999. There is probably a way to make sure it's always on top, but I'm starting to step outside my realm of experience.